src/runner.py

```diff
@@ -6,10 +6,12 @@ def __init__(
         self,
         model=None,
         device=None,
+        input_key=None,
```

Comment:
so, since you pass `runner_args` to the config, you can easily remove this custom `Runner` class

and in `__init__.py` use `from catalyst.dl import SupervisedRunner as Runner`. That's all 🙂

Reply:
done